examples/guestbook/v1beta3/redis-master-service.json

```diff
@@ -0,0 +1,18 @@
+{
+    "kind": "Service",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "redis-master",
+        "labels": {
+            "name": "redis-master"
```

Comment:
As discussed, we can fix this when fixing (or removing) the original v1beta1 example.